docs/releases/1.8.5.txt

```diff
@@ -26,3 +26,5 @@ Bugfixes
 
 * Fixed autocompletion for options of non-``argparse`` management commands
   (:ticket:`25372`).
+
+* Fixed ordering of imports in newly created migrations (:ticket:`25384`).
```

Comment:
"Alphabetized ordering of imports in `from django.db import migrations, models` statement in ..." might be more clear.